main_test.go

```diff
@@ -162,32 +217,58 @@ var _ = Describe("Main", func() {
 						ghttp.VerifyRequest("GET", "/v2/organizations/"+orgGuid),
 						ghttp.RespondWithJSONEncoded(http.StatusOK, testOrgResource(orgGuid)),
 					),
+					ghttp.CombineHandlers(
+						ghttp.VerifyRequest("GET", "/v2/apps"),
+						ghttp.RespondWithJSONEncoded(http.StatusOK, testAppResponse(afterApps)),
+					),
+					ghttp.CombineHandlers(
```

Comment:
This is duplicated code, from here to the end of the argument list. I think we can remove it all, because the HTTP mocks are already in place.

Reply:
superfluous 🙌